commonTools/build_stats/unit_tests/build_stats.big.small.csv

```diff
@@ -0,0 +1,22 @@
+avg_size_unshared_text_area_Kb,max_resident_size_Kb,cpu_sec_user_mode,elapsed_real_time_sec,num_involuntary_context_switch,FileName,symbol_text_local,FileSize,cpu_sec_kernel_mode,num_filesystem_outputs

```

Comment:
Should this csv file be included? Looks like it is output from a run that was accidentally included.

Reply:
@rppawlo, this file gets used in some automated system-level tests of the `summarize_build_stats.py` tool.  See:

https://github.com/trilinos/Trilinos/blob/afca3be542dd0b9955ea138070e31d89e29808c1/commonTools/build_stats/unit_tests/summarize_build_stats_UnitTests.py#L64